app/src/components/Layout.tsx

```diff
@@ -1,26 +1,32 @@
+import { Grid, GridContainer } from "@trussworks/react-uswds";
 import { useTranslation } from "next-i18next";
-import { ReactElement } from "react";
+
+import Footer from "./Footer";
+import Header from "./Header";
 
 type Props = {
-  children: ReactElement;
+  children: React.ReactNode;
 };
 
-const Layout = ({ children }: Props): ReactElement => {
-  const { t } = useTranslation("common");
+const Layout = ({ children }: Props) => {
+  const { t } = useTranslation("common", {
+    keyPrefix: "Layout",
+  });
 
   return (
-    <div>
-      <header /* add your agency's or USWDS's header here */>
-        <em>{t("Layout.header")}</em>
-      </header>
-      <main className="grid-container">
-        <div className="grid-row">
-          <div className="grid-col">{children}</div>
-        </div>
+    <div className="display-flex flex-column minh-viewport">
```

Comment:
ℹ️ Sticks the footer to the bottom of the page

Reply:
useful to add a code comment?

Reply:
Added 👍🏻